x-pack/docs/en/security/limitations.asciidoc

```diff
@@ -51,7 +52,7 @@ When a user's role enables document or field level security for a data stream or
 ** Update requests included in bulk requests aren't supported.
 * The request cache is disabled for search requests.
 
-When a user's role enables document level security for a data stream or index:
+When a user's role enables <<document-level-security,document level security>> for a data stream or index:
```

Comment:
👍 I think we can also cross link to Field Level Security in the previous paragraph.